functions/common.py

```diff
@@ -16,11 +16,11 @@ def factorial(argument: int) -> int:
     """
     # Ensure that input is an integer.
     if int(argument) != argument:
-        raise exceptions.InputError(argument, "Input to factorial should be a non-negative integer.")
+        raise exceptions.InputError(argument, "Input to factorial must be a positive integer.")
```

Comment:
I'm a little late to this PR but the idea here was that 0 is not a positive integer. Typically, positive integer refers to all integers greater than 0 and non-negative integer refers to all integers greater than or equal to 0.